docs/api/utils.md

```diff
@@ -244,75 +244,51 @@ const todoFamily = atomFamily(
 
 https://codesandbox.io/s/react-typescript-forked-8zfrn
 
-## useSelector
+## selectAtom
 
 Ref: https://github.com/pmndrs/jotai/issues/36
 
-### Usage
-
 ```js
-useSelector(anAtom, selector, equalityFn)
+function selectAtom<Value, Slice>(
+  anAtom: Atom<Value>,
+  selector: (v: Value) => Slice,
+  equalityFn: (a: Slice, b: Slice) => boolean = Object.is
+): Atom<Slice>
 ```
 
-Selector and equalityFn must be stable (should be wrapped with useCallback).
-The equalityFn is optional.
+This function creates a derived atom whose value is a function of the original atom's value,
+determined by `selector.`
+The selector function runs whenever the original atom changes; it updates the derived atom
+only if `equalityFn` reports that the derived value has changed.
+By default, `equalityFn` is reference equality, but you can supply your favorite deep-equals
+function to stabilize the derived value where necessary.
 
 ### Examples
 
-```jsx
-import { Provider } from 'jotai'
-import { useSelector, atomWithReducer, useUpdateAtom } from 'jotai/utils'
-
-const initialState = {
-  count: 0,
-  text: 'hello',
-}
-
-const reducer = (state, action) => {
-  if (action.type === 'INC') {
-    return { ...state, count: state.count + 1 }
-  } else if (action.type === 'SET_TEXT') {
-    return { ...state, text: action.text }
-  } else {
-    throw Error('no such action')
-  }
+```js
+const defaultPerson = {
+  name: {
+    first: 'Jane',
+    last: 'Doe',
+  },
+  birth: {
+    year: 2000,
+    month: 'Jan',
+    day: 1,
+  },
 }
 
-const stateAtom = atomWithReducer(initialState, reducer)
+// Original atom.
+const person = atom(defaultPerson)
 
-const selectCount = (state: State) => state.count
+// Tracks person.name; updated when person.name gets a new reference
+const name = selectAtom(defaultPerson, (person) => person.name)
 
-const Counter = () => {
-  const dispatch = useUpdateAtom(stateAtom)
-  const count = useSelector(stateAtom, selectCount)
-  return (
-    <div>
-      {count} <button onClick={() => dispatch({ type: 'INC' })}>+1</button>
-    </div>
-  )
-}
-
-const selectText = (state: State) => state.text
-
-const TextBox = () => {
-  const dispatch = useUpdateAtom(stateAtom)
-  const text = useSelector(stateAtom, selectText)
-  return (
-    <div>
-      {text}{' '}
-      <input
-        value={text}
-        onChange={(e) => dispatch({ type: 'SET_TEXT', text: e.target.value })}
-      />
-    </div>
-  )
-}
+// Tracks person.birth; updated when year, month, or day change
+import deepEqual from 'fast-deep-equal'
+const age = selectAtom(defaultPerson, (person) => person.birth, deeqEqual)
```

Comment:
```suggestion
const ageAtom = selectAtom(defaultPerson, (person) => person.birth, deeqEqual)
```

Actually, for this specific example, shallowEqual is enough. maybe it's fine as is.